### DISCUSSION TITLE
🔗 Ability to disable the icon in the WooCommerce Notices widget (Related to #25709)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#25710] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

It would be great if you add this feature in WooCommerce Notices so we can disable the icon in it. This is a simple but useful feature.

![a](https://user-images.githubusercontent.com/88034902/213911280-475be8b2-77c7-486c-b895-061d6e337ac4.png)

### Describe the solution you'd like

A new option to disable icons in WooCommerce Notices.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #21091 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by nicholaszein** [comment]
This Request was mentioned in the following issues and discussions:

- #25709
